libraries/exporters/src/main/java/org/eclipse/sw360/exporter/helper/ReleaseHelper.java

```diff
@@ -134,6 +134,11 @@ private void addFieldValueToRow(List<String> row, Release._Fields field, Release
                     row.add("");
                 } else {
                     row.add(ThriftEnumUtils.enumToString(component.getComponentType()));
+                    if (component.getCategories() == null) {
+                        row.add("");
+                    } else {
+                        row.add(component.getCategories().toString());
```

Comment:
Is it possible that categories are null so that you can get a NPE here? Probably not because I had a component without categories and it did not break - if I remember correctly. But I am not sure, if this is guaranteed.

Reply:
Yes.I tried to create a component without setting the categories field and when I linked any of the release to project from the same category then in that case application crashes.

Reply:
Wait, it crashed or it didn't crash? If it crashed then you should probably do a null check before you call `toString()` on `getCategories()`.

Reply:
Sorry, my bad, you already added the null check!